code/datums/components/butchering.dm

```diff
@@ -161,3 +162,54 @@
 		return
 	if(victim.stat == DEAD && (victim.butcher_results || victim.guaranteed_butcher_results))
 		Butcher(parent, victim)
+
+/datum/component/butchering/wearable
+
+/datum/component/butchering/wearable/RegisterWithParent()
+	. = ..()
+	RegisterSignal(parent, COMSIG_MECHA_EQUIPMENT_ATTACHED, .proc/enable_butchering)
+	RegisterSignal(parent, COMSIG_MECHA_EQUIPMENT_DETACHED, .proc/disable_butchering)
+	RegisterSignal(parent, COMSIG_ITEM_EQUIPPED, .proc/worn_enable_butchering)
+	RegisterSignal(parent, COMSIG_ITEM_DROPPED, .proc/worn_disable_butchering)
+	RegisterSignal(parent, COMSIG_MECHA_DRILL_MOB, .proc/Butcher)
```

Comment:
can you lowercase to ``butcher``? Even better if you do ``on_drill``

Reply:
wouldnt on_butchering be better since the proc is also being used by recycler?